contrib/fuzz/metadata_fuzzer.go

```diff
@@ -0,0 +1,407 @@
+// +build gofuzz
+
+/*
+   Copyright The containerd Authors.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+// nolint: golint
```

Comment:
golint is helpful. What are the errors you'd like to ignore?

Reply:
Removed `nolint` and all tests pass. I can't say why that test failed last time.